res/Sectors/M1105/Muarne.xml

```diff
@@ -0,0 +1,61 @@
+﻿<Sector>
+  <X>5</X>
+  <Y>6</Y>
+  <Name Lang="hv">Muarne</Name>
+  <DataFile Type="SecondSurvey">Muarne.txt</DataFile>
```

Comment:
For future reference: don't put <DataFile> elements into the linked XML files. If it gets out of sync between the milieu XML file and the sector XML file the site gets sad.